src/components/features/NoticeTree/MessageBox/index.tsx

```diff
@@ -9,11 +9,18 @@ const MessageBox = ({ selectedMessage, showMessageHandler }: MessageBoxProps) =>
 			{selectedMessage && (
 				<S.MessageBoxWrapper>
 					<S.MessageBoxInner>
-						<S.MessageText> {selectedMessage.message} </S.MessageText>
+						<S.MessageText> {selectedMessage.content} </S.MessageText>
 						<S.CancleIcon onClick={() => showMessageHandler(false)}>X</S.CancleIcon>
 						<S.MessageWriterWrapper>
-							<S.MessageWriterImage src={AlertBee} alt="messageWriterImage" />
-							<S.MessageWriter> {selectedMessage.writer ? selectedMessage.writer : '익명의 꿀벌'}</S.MessageWriter>
+							<S.MessageWriterImage
+								src={
+									selectedMessage.senderProfileImage.includes('http') ? selectedMessage.senderProfileImage : AlertBee
+								}
+								alt="messageWriterImage"
+							/>
+							<S.MessageWriter>
+								{selectedMessage.senderNickname ? selectedMessage.senderNickname : '익명의 꿀벌'}
```

Comment:
```suggestion
								{selectedMessage.senderNickname ||  '익명의 꿀벌'}
```

논리 연산자(`||`) 로 처리하면 좀 더 간결하지 않을까 의견드려봅니다 ~!